Description: 900px is too tall. Not all window managers will resize such a window. Use 700.
Author: Iain Lane <iain.lane@canonical.com>
Origin: upstream, dd6c6bd4df9cf71f6e4875125760c1700067fa2d

Index: b/src/disks/gduwindow.c
===================================================================
--- a/src/disks/gduwindow.c
+++ b/src/disks/gduwindow.c
@@ -1275,8 +1275,7 @@
 
   gtk_widget_reparent (window->main_hpane, GTK_WIDGET (window));
   gtk_window_set_title (GTK_WINDOW (window), _("Disks"));
-  /* Fortunately the window manager resizes the window if it's too tall for the monitor.. so 900 pixels is fine */
-  gtk_window_set_default_size (GTK_WINDOW (window), 800, 900);
+  gtk_window_set_default_size (GTK_WINDOW (window), 800, 700);
   gtk_window_set_position (GTK_WINDOW (window), GTK_WIN_POS_CENTER);
   gtk_container_set_border_width (GTK_CONTAINER (window), 12);
 
